src/Keywords/CucumberKeywords.php

```diff
@@ -50,6 +50,10 @@ public function __construct($yaml)
             throw $e;
         }
 
+        if (!is_array($content)) {
+            throw new ParseException(sprintf('Root element must be an array, but %s found.', get_debug_type($content)));
```

Comment:
This a bit better than getting a `TypeError` from calling the parent constructor which requires an array value.